codecov.yml

```diff
@@ -78,16 +78,17 @@ comment:
   # This is the addition of carry forward builds and fresh builds, thus, it's the addition
   # of the FE and BE builds
   after_n_builds: 22
-  layout: 'diff, files'
-  # Update, if comment exists. Otherwise post new.
-  behavior: default
-  # Comments will only post when coverage changes. Furthermore, if a comment
-  # already exists, and a newer commit results in no coverage change for the
-  # entire pull, the comment will be deleted.
-  require_changes: true
+  # Show only the git diff / patch coverage, and exclude any comparison between base and head:
+  layout: 'condensed_header, condensed_files, condensed_footer'
+  hide_project_coverage: true
```

Comment:
These two lines will enable the following:
![image](https://github.com/user-attachments/assets/c24159d5-8840-4f0d-8df5-c092c72f4312)